pywbem/mof_compiler.py

```diff
@@ -1611,14 +1613,18 @@ def p_instanceDeclaration(p):
                           'Duplicate property: %s' % pname)
             ce.file_line = (p.parser.file, p.lexer.lineno)
             raise ce
-
         try:
             # build instance property from class property but without
             # qualifiers, default value,
             pprop = cprop.copy()
             pprop.qualifiers = NocaseDict(None)
             pprop.value = tocimobj(cprop.type, pval)
             inst.properties[pname] = pprop
+            # if alias and this is key property, add keybinding
+            keybindings = NocaseDict()
```

Comment:
The `keybindings` dictionary should probably be initialized before the loop through the properties.